configv2/configv2.go

```diff
@@ -248,7 +249,7 @@ func loadExporters(v *viper.Viper) (models.Exporters, error) {
 		}
 
 		// Find exporter factory based on "type" that we read from config source
-		factory := factories.GetExporterFactory(typeStr)
+		factory := exporter.GetExporterFactory(typeStr)
```

Comment:
nit: a rename seems good: `exporter.GetFactory(typeStr)`

Likely a symmetrical change applies to receivers, etc.

Reply:
Yes, I wanted to do exactly this, but in a future PR. :-)

Reply:
👍